test/FieldVector.jl

```diff
@@ -18,19 +18,21 @@
         @test length(p) === 3
         @test eltype(p) === Float64
 
+        @test (p + p)::Point3D ≈ Point3D(2.0, 4.0, 6.0)
```

Comment:
The rules say that basic floating point arithmetic operations should be correctly rounded.  In particular, this means that if *all intermediate results* of a computation are exactly representable, then the final result is exact.

This certainly holds true when you're adding, subtracting and multiplying exact integers using floating point arithmetic, up until `maxintfloat(Float64) == 9.007199254740992e15`.

The reason I bring this up is that `isapprox` uses quite a generous default for `rtol`, a lot looser than you'd really want in many tests.